cpp/cmake_modules/SetupCxxFlags.cmake

```diff
@@ -496,14 +494,24 @@ if(ARROW_CPU_FLAG STREQUAL "armv8")
     endif()
     set(CXX_COMMON_FLAGS "${CXX_COMMON_FLAGS} ${ARROW_ARMV8_ARCH_FLAG}")
 
+    set(ARROW_HAVE_NEON ON)
```

Comment:
Is NEON always available with SVE?

Reply:
Yes, NEON is always available for aarch64.